### PR TITLE
[Doc] add seatunnel engine to start-v2

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

**Review by Hisoka-X — DISMISSED** [review]

File: docs/en/start-v2/local.mdx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+connector-fake
+connector-console
+--end--
+```
+
+You can find all supported connectors and corresponding plugin_config configuration names under `${SEATUNNEL_HOME}/connectors/plugins-mapping.properties`.
+
+:::tip
+
+If you want to install the connector plugin by manually downloading the connector, you need to pay special attention to the following
```
Comment (ic4y) [inline docs/en/start-v2/local.mdx:64]: following what？

File: docs/en/start-v2/local.mdx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+<!-- TODO: We should add example module as quick start which is no need for install Spark or Flink -->
+
+## Step 3: Install connectors plugin
+Since 2.3.0-beta, the binary package does not provide connector dependencies by default, so when using it for the first time, we need to execute the following command to install the connector: (Of course, you can also manually download the connector from [Apache Maven Repository](https://repo.maven.apache.org/maven2/org/apache/seatunnel/ to download, then manually move to the seatunnel subdirectory under the connectors directory).
+```bash
+sh bin/install_plugin.sh 2.3.0-beta
+```
+If you need to specify the version of the connector, take 2.3.0-beta as an example, we need to execute
+```bash
+sh bin/install_plugin.sh 2.3.0-beta
```
Comment (ic4y) [inline docs/en/start-v2/local.mdx:43]: These two commands are the same?

File: docs/en/start-v2/local.mdx (hunk 1 above)
Comment (EricJoy2048) [reply docs/en/start-v2/local.mdx:64]: done

**Review by hailin0 — DISMISSED** [review]

**Review by Hisoka-X — DISMISSED** [review]

**Comment by Hisoka-X** [comment]
Please modify `sidebars.js`

File: docs/sidebars.js [hunk 3]
```diff
@@ -44,173 +44,186 @@ const sidebars = {
...
         },
-        'Connector-v2-release-state',
         {
-            type: 'category',
-            label: 'Connector-V2',
-            items: [
+            "type": "category",
+            "label": "Concept",
+            "items": [
+                "concept/config"
```
Comment (Hisoka-X) [inline docs/sidebars.js]: Miss 'concept/connector-v2-features'

**Review by Hisoka-X — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**